### PR TITLE
[W5.11][F12-A4] Cheng Wenhao

### PR DESCRIPTION
**Pull request by wenhaoch**
Added line breaks in user guide for better readability
Added case-insensitive sorting functionality

### CODE REVIEW

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (KevinChuangCH) [inline src/seedu/addressbook/parser/Parser.java]: I think the imports should be listed explicitly according to the codding standard.

Comment (wenhaoch) [reply src/seedu/addressbook/parser/Parser.java]: Right. I must have missed it out as I was resolving the conflicts while adding the code. Thanks!

File: src/seedu/addressbook/commands/HelpCommand.java [hunk 2]
```diff
@@ -16,15 +16,15 @@ public HelpCommand() {}
...
-                + "\n" + ExitCommand.MESSAGE_USAGE
+                "\n" + AddCommand.MESSAGE_USAGE
+                + "\n\n" + DeleteCommand.MESSAGE_USAGE
+                + "\n\n" + ClearCommand.MESSAGE_USAGE
+                + "\n\n" + FindCommand.MESSAGE_USAGE
+                + "\n\n" + ListCommand.MESSAGE_USAGE
+                + "\n\n" + ViewCommand.MESSAGE_USAGE
+                + "\n\n" + ViewAllCommand.MESSAGE_USAGE
+                + "\n\n" + HelpCommand.MESSAGE_USAGE
+                + "\n\n" + ExitCommand.MESSAGE_USAGE
```
Comment (KevinChuangCH) [inline src/seedu/addressbook/commands/HelpCommand.java]: Greatly improved the readability! Nice refactoring.

**Review by OuyangDanwen — COMMENTED** [review]
Well done for completing all requirements and writing generally neat code. Take note that commit messages should be written in imperative, i.e. always start with a verb in present tense. Please close the PR after reading the comments. @wenhao53

**Pull request closed without merging**